components/general/ContactPage.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from "@/styles/ContactPage.module.css";
```

Comment:
what is the point of this component?  why this is not in the contact,tsx?

Reply:
Cause, one of the previous sections has a component already called Contact.tsx.